cmd/orchard-sidebar/main.go

```
@@ -0,0 +1,659 @@
+	}
+	home, _ := os.UserHomeDir()
+	return home + "/.local/state/claude-sessions/state"
+}
+
+func pidAlive(pid int) bool {
+	return pid > 0 && syscall.Kill(pid, 0) == nil
+}
+
+// paneToSession maps tmux pane ids (%5) to session names via one tmux call.
+func paneToSession() map[string]string {
+	out, err := exec.Command("tmux", "list-panes", "-a", "-F", "#{pane_id} #{session_name}").Output()
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)